backend/sql/initial_schema.sql

```diff
@@ -1,11 +1,11 @@
-create table project
+create table if not exists project
```

Comment:
> we didn't clean up it properly previous time (like there is a forgotten table in drop.sql)

@pva701 I don't understand, why do we want to clean up? Are you talking about developer workflow? I mean in the normal production workflow we don't need `drop.sql`, right? We run the production server, restart it for some reason (maybe the computer just died for example) and we actually want to `create table if not exists` in this case, don't we?

Reply:
I meant developer workflow right/.

>we actually want to create table if not exists in this case, don't we?

I don't know, you told himself that in production workflow we create db once, so why do we need this check `if not exists`?

Reply:
I mean, this db creation from code on every run seems not to be done in normal flow. I mean, this is just an ugly hack for development which resided in disciplina/agora which considered de-facto standard in our web apps.

But what is the purpose of this code to be run on production? I mean, every if we take into consideration migrations, they rather will check some last applied sql migration and try to apply next, so no relying on existing of tables, it's just nonsense.

This code should be run only in dev mode IMHO, and in devmode as schema might be changed often, it should be robust, but checking that there exists table with the same name doesn't seem robust to me (in most cases it will be previous version of table if the db was forgotten to be dropped).